Applications/Spire/Include/Spire/Ui/FlatButton.hpp

```diff
@@ -34,7 +34,9 @@ namespace Spire {
 
       //! Constructs the FlatButton.
       /*!
-        \param label The label text.
+        \param label The label text. If the label text is empty, the button
+                     is considered a color button and is drawn with white
+                     padding.
         \param parent The parent widget to the FlatButton.
       */
       explicit FlatButton(const QString& label, QWidget* parent = nullptr);
```

Comment:
You have a ```label``` parameter here, but a ```set_text``` method later. Rename that method to ```set_label```. Also ensure that if that method is given an empty label that it also behaves like a "color" button.